templates/hound.yml

```diff
@@ -1,17 +1,17 @@
 # See https://houndci.com/configuration for help.
 coffeescript:
-  # config_file: .coffeescript-style.json
+  config_file: .coffeescript-style.json
   enabled: true
 haml:
   # config_file: .haml-style.yml
   enabled: true
 javascript:
-  # config_file: .javascript-style.json
+  config_file: .javascript-style.json
   enabled: true
   # ignore_file: .javascript_ignore
 ruby:
-  # config_file: .ruby-style.yml
+  config_file: .ruby-style.yml
```

Comment:
The Ruby one should definitely be `.rubocop.yml`, as that's where Rubocop looks, and Ruby developers will think to look there.